<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_popup_Purchase_Job_Contract</class>
 <widget class="QMainWindow" name="frm_popup_Purchase_Job_Contract">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>合同</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_no">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>合同编号</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>合同名称</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_PartyA">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>甲方客户</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_PartyB">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>乙方客户</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_amount">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>合同金额</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_status">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>250</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>合同状态</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_no">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_PartyA">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_amount">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_name">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_PartyB">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_status">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_Cancel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_Save">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_PartyA_Pick">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>☺</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_PartyB_Pick">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>150</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>☺</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>626</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
